### DISCUSSION TITLE
vue.js和svelte.js可以在modern.js体系里使用吗

### DISCUSSION THREAD

**Discussion by NeroBlackstone** [web-infra-dev/modern.js#244] [Q&A] [answered]
我在文档里看好像只能写react

**Comment by chenjiahan** [comment, marked as answer]
Modern.js Framework 是基于 React 的，不支持 Vue 和 Svelte 呢。

不过我们后续计划在 Modern.js Builder 中支持 Vue 项目的构建，提供一些相对基础的支持

**Reply by chenjiahan** [reply]
Update: use Rsbuild https://github.com/web-infra-dev/rsbuild
